packages/graphql/src/translate/create-create-and-params.ts

```diff
@@ -89,6 +89,20 @@ function createCreateAndParams({
                 const unionTypeName = relationField.union || relationField.interface ? refNode.name : "";
 
                 if (v.create) {
+                    const isInterfaceAnArray = relationField.interface?.typeMeta.array;
+                    const createNodeInputIsOfTypeRefNode = !!v.create.node?.[refNode.name];
+                    const createNodeInputKeys = createNodeInputIsOfTypeRefNode
+                        ? Object.keys((v.create.node as any[]) || [])
+                        : [];
+                    const isCreatingMultipleNodesForOneToOneRel = !isInterfaceAnArray && createNodeInputKeys.length > 1;
+                    if (isCreatingMultipleNodesForOneToOneRel) {
+                        throw new Error(
+                            `Relation field "${
+                                relationField.interface?.dbPropertyName || relationField.interface?.fieldName
+                            }" cannot have more than one node linked`
```

Comment:
Just a suggestion about error message wording here, would be good to also have the typename in the error message. 🙂 

```suggestion
                            `Relationship field "${node.name}.${
                                relationField.interface?.dbPropertyName || relationField.interface?.fieldName
                            }" cannot have more than one node linked`
```